common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -18,6 +19,7 @@ import java.io.OutputStreamWriter
 import java.net.HttpURLConnection
 import java.net.MalformedURLException
 import java.net.URL
+import kotlin.jvm.Throws
```

Comment:
Are you using this?

Reply:
ah, the file said "unresolved reference: Throws", but the sample app still runs without it. I'll remove this